digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -239,7 +239,7 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
         case RoadWidth.typeId => roadWidthService
         case HazmatTransportProhibition.typeId | Prohibition.typeId => prohibitionService
         case EuropeanRoads.typeId | ExitNumbers.typeId => textValueLinearAssetService
-        case CareClass.typeId | DamagedByThaw.typeId  => dynamicLinearAssetService
+        case CareClass.typeId | DamagedByThaw.typeId | AnimalWarnings.typeId  => dynamicLinearAssetService
```

Comment:
Do you know why `massTransitLanes `is not mapped to the `dynamicLinearAssetService`?

Reply:
It's not mapped to dynamicLinearAssetService because they have her own method to map to the Api, massTransitLanesToApi, this is only used for the massTransitLanes assets and that, uses directly the dynamicLinearAssetService on the method getMultiValueLinearAssetByMunicipality.

Happens the same for carryingCapacities asset.

Was removed at 20/07/2018 related with DROTH-1361 (US 341) Database: Add new asset type and properties.